test/integration/create-next-app/templates-app.test.ts

```diff
@@ -13,12 +13,10 @@ import {
   shouldBeTemplateProject,
   spawnExitPromise,
 } from './lib/utils'
-//import { Span } from 'next/dist/trace'
 
 import { useTempDir } from '../../lib/use-temp-dir'
 import { fetchViaHTTP, findPort, killApp, launchApp } from 'next-test-utils'
 import resolveFrom from 'resolve-from'
-//import { createNextInstall } from '../../lib/create-next-install'
```

Comment:
Similarly here - was this intended?

Reply:
Yeah no need to keep the comment around as we have the TODO